lib/Dialect/SV/Transforms/HWMemSimImpl.cpp

```diff
@@ -62,7 +62,7 @@ class HWMemSimImpl {
 
   Value addPipelineStages(ImplicitLocOpBuilder &b,
                           ModuleNamespace &moduleNamespace, size_t stages,
-                          Value clock, Value data, Value gate = {});
+                          Value clock, Value data, Twine name, Value gate = {});
```

Comment:
```suggestion
                          Value clock, Value data, const Twine & name, Value gate = {});
```

By-value Twine is asking for trouble (and doxygen says to always use as const ref).  Whoops too late.

Reply:
Good catch. I'll push a fix.

Reply:
Fixed in https://github.com/llvm/circt/commit/f0e9e99cef738fca40c5ad51460c38f668ee33a6. Thanks!